src/ssl.rs

```diff
@@ -0,0 +1,69 @@
+use std::io::{self, ErrorKind};
+use std::net::{SocketAddr, TcpStream};
+
+use stream::Stream;
+
+use openssl::ssl::{Ssl, SslMethod, SslContext, SslStream, SSL_OP_NO_COMPRESSION, SSL_OP_NO_SSLV2,
+                   SSL_OP_NO_SSLV3, SSL_VERIFY_NONE, SSL_VERIFY_PEER};
+use openssl::x509::X509_FILETYPE_PEM;
+
+#[derive(Clone, Debug)]
+pub struct SslConfig {
```

Comment:
Actually, this file isn't supposed to even exist anymore. As mentioned below, I totally meant to delete it.